### ISSUE TITLE
Unicode characters are escaped in requests from Slack

### ISSUE THREAD

**Issue by michaelb990** [socrata/SlackPoll#15]
This causes an error when printing logs: line 22 in slack-poll.py

**Comment by courtneyspurgeon** [comment]
Is this also what causes a 500 error when questions contain apostrophes, or is that a new issue?

Repro: `/poll create What’s up? options Yes | No`

**Comment by michaelb990** [comment]
Yep, I'm pretty sure that's why. It was getting escaped to: `/poll create
What\u344s up? options Yes | No` Or whatever the unicode code is for the
apostrophe. Should be a pretty easy fix.

On Fri, Nov 20, 2015 at 12:34 PM, courtneyspurgeon <notifications@github.com
